core/src/builtins/mod.rs

```diff
@@ -46,6 +46,14 @@ impl Builtins {
 
         builtin!("./console.js");
         builtin!("./queue_microtask.js");
+        let source = include_str!("./url-bundle.js");
+        let _val = match crate::script::run(scope, source, "./url-bundle.js") {
+            Ok(v) => v,
+            // if this in unreachable it fails
+            // Err(_) => unreachable!(),
+            // if we ignore error it is fine
```

Comment:
There is an error being thrown by run during build for snapshots that I can't figure out, might have something to do with the generated rollup bundle.

Swallowing error (along with disabling snapshot) is enough to show that jstime without snapshots can run the bundled output being generated from whatwg-url